executor/cook/config.py

```diff
@@ -31,13 +32,15 @@ def parse_time_ms(time_string):
     def __init__(self,
                  max_bytes_read_per_line=1024,
                  max_message_length=512,
+                 progress_output_env_variable='EXECUTOR_PROGRESS_OUTPUT_FILE',
```

Comment:
This default value is never used.

Reply:
It is used in `test_retrieve_process_environment`

Reply:
Should we create a constant for `'EXECUTOR_PROGRESS_OUTPUT_FILE'` since we use it here and below?